CMakeLists.txt

```diff
@@ -102,7 +102,8 @@ endif()
 
 #Switch off modern thread local for dmlc-core, please see: https://github.com/dmlc/dmlc-core/issues/571#issuecomment-543467484
 add_definitions(-DDMLC_MODERN_THREAD_LOCAL=0)
-
+# disable stack trace in exception by default.
+add_definitions(-DDMLC_LOG_STACK_TRACE_SIZE=0)
```

Comment:
Don't we want some amount of stack traces for users having MXNet in production and experiencing problems.

Reply:
Where is this being used? Also the environment variable has a similar but different name.

Reply:
It's an existing variable in dmlc-core. It can be explicitly turned on through environment variable.